docs/examples/pydata.ipynb

```diff
@@ -30,6 +35,46 @@
     "df"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## IPyWidget"
```

Comment:
I edited this example notebook to add a pandas data table inside of an ipywidget to help us catch issues like #1740 in the future.